Pgvector.sln

```diff
@@ -5,24 +5,24 @@ VisualStudioVersion = 17.0.31903.59
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{B76AB67C-094F-433B-AF25-807B50D2888E}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Pgvector", "src\Pgvector\Pgvector.csproj", "{3C580CE2-B935-4656-BCC9-4FB4035089E4}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Pgvector", "src\Pgvector\Pgvector.csproj", "{3C580CE2-B935-4656-BCC9-4FB4035089E4}"

```

Comment:
All these sln changes seem unrelated and probably shouldn't be in this PR?

Reply:
Will be reverted